med_app_mobile/lib/pages/registration_page.dart

```diff
@@ -0,0 +1,181 @@
+                            decoration: ThemeHelper().textInputDecoration('Password', 'Enter your password'),
+                          ),
+                          decoration: ThemeHelper().inputBoxDecorationShaddow(),
+                        ),
+                        const SizedBox(height: 30),
+                        Container(
+                          // Tu trzeba sprawdzić czy hasło takie samo jak poprzednie
+                          child: TextField(
+                            obscureText: true,
+                            decoration: ThemeHelper().textInputDecoration('Confirm password', 'Confirm your password'),
+                          ),
+                          decoration: ThemeHelper().inputBoxDecorationShaddow(),
+                        ),
+                        FormField<bool>(
+                          builder: (state) {
+                            return Column(
+                              children: <Widget>[
+                                Row(
+                                  children: <Widget>[
+                                    Checkbox(
+                                      value: checkboxValue.value,
+                                      onChanged: (value) {
+                                        setState(() {
+                                          checkboxValue.value = value;
+                                          state.didChange(value);
+                                        });
+                                      },
+                                    ),
+                                    const Text(
+                                      'I accept all terms and conditions',
+                                      style: TextStyle(
+                                        color: Colors.black,
+                                      ),
+                                    ),
+                                  ],
+                                ),
+                                Container(
+                                  alignment: Alignment.center,
+                                  child: Text(
+                                    state.errorText ?? '',
+                                    textAlign: TextAlign.center,
+                                    style: TextStyle(
+                                      color: Theme.of(context).errorColor,
+                                      fontSize: 12
+                                    ),
+                                  ),
+                                ),
+                              ],
+                            );
+                          },
+                          validator: (value) {
+                            // Tutaj IF który sprawdzi czy Terms&Cond są zaznaczone
+                          },
+                        ),
+                        const SizedBox(width: 20),
+                        Container(
+                          // To jest przycisk tworzący konto, podpiać bezę danych
```

Comment:
bezę to moja Mama nauczyła mnie robić. Całkiem niezła, z mascarpone i malinami. A jak ją tu podepniecie?